core/jobs/batch_jobs/state_interaction_validation_jobs.py

```diff
@@ -532,7 +532,24 @@ def filter_invalid_fraction_input_interaction(
                             rule_spec.rule_type == (
                                 'IsEquivalentToAndInSimplestForm')
                         ):
-                            rule_value_f = int(rule_spec.inputs['f'])
+                            rule_value_f = rule_spec.inputs['f'].strip()
+                            if '/' in rule_value_f:
+                                rule_value_f = rule_value_f.split('/')
+                                # For values like '1 3/2'
+                                if len(rule_value_f[0].strip().split()) > 1:
+                                    value_1 = rule_value_f[0].strip().split()
+                                    value_2 = rule_value_f[1]
+
+                                    rule_value_f = (((float(value_2)*float(
+                                        value_1[0]))+ float(
+                                            value_1[1])) / float(value_2))
```

Comment:
Shouldn't `1 3/2` end up being calculated as `1 + 3/2`? So `value_1[0] + value_1[1] / value_2`?

Reply:
Yep!, I was solving the whole value. Edited